apps/api/src/coscrad-cli/ingest-media-items.cli-command.ts

```diff
@@ -106,7 +106,6 @@ const buildCreateResourceFsaForMediaItem = (
                 languageCodeForTitle: LanguageCode.English,
                 mediaItemId,
                 // TODO What should we do about this? Maybe it is indicated in the filename? The real metadata? Should be optional?
```

Comment:
I'm trying to remember the original discussion. Did we want to allow for photographers who are not registered as contributors? If not, just simply use the `contributorIds` on the creation event meta.

Reply:
Right, I guess there's a few more aspects to the photographer property.  In some cases people won't know the photographer, like in the case of historical/archival photos.  Would it be too much complexity to add a role to the contributor model?  With options like "Photographer", "Author", "Storyteller", "Elder Advisor", "Language Speaker", "Artist", "Knowledge Keeper", etc.  If that were possible then we could just use the contributor.

Reply:
I don't agree with adding a role to the contributor model for this purpose. The role is inferred from the action(s) that contributed to the event history. If someone "CREATED_PHOTOGRAPH", effectively, they are a photographer, among other roles. However, I can see a case where groups don't want the photographer to be listed as a contributor. This is where just having a `photographer: string` might have been an ok work around. What are your thoughts on that?

Reply:
That said, it's fine to categorize the contributor in their bio as a "knowledge keeper" or "photographer", but I think the event history should be the source of truth. I don't want to maintain a separate layer \ enum for these. It seems like added artificial complexity.

Reply:
Ok right, yeah I guess it makes sense to go back to having photographer.  I agree with the idea of keeping the events as the source of truth, but then there will be many cases where someone uploading and causing a "CREATED_PHOTOGRAPH" event to trigger will just be a regular user and not the person who took the photograph.

Reply:
Sure. Remember that there **is** a distinction between `event.meta.userId` (the one who is entering the data) and `event.meta.contributorIds`, the ones who are responsible for the actual content or work. But I think keeping `photographer` for now will sidestep the issue.